processor/configuration/src/test/java/com/redhat/service/smartevents/processor/resolvers/AbstractGatewayValidatorTest.java

```diff
@@ -31,11 +31,14 @@ protected static Action actionWith(String type, Map<String, String> params) {
     protected void assertValidationIsInvalid(Gateway gateway, String errorMessage) {
         ValidationResult validationResult = getValidator().isValid(gateway);
         assertThat(validationResult.isValid()).isFalse();
-        assertThat(validationResult.getMessage()).startsWith(errorMessage);
+        assertThat(validationResult.getMessage()).containsAnyOf(errorMessage);
     }
 
     protected void assertValidationIsValid(Gateway gateway) {
         ValidationResult validationResult = getValidator().isValid(gateway);
+        if (validationResult.getMessage() != null) {
+            assertThat(validationResult.getMessage()).isEmpty();
+        }
```

Comment:
Do you need this `null` check? As far as I remember `assertThat(..)` is able to handle this scenario directly. Can you please verify?
```suggestion
        assertThat(validationResult.getMessage()).isEmpty();
```